### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: nwc/src/main/webapp/js/view/AquaticBiologySelectFeaturesView.js [hunk 1]
```diff
@@ -25,8 +25,9 @@ NWC.view.AquaticBiologySelectFeaturesView = NWC.view.BaseView.extend({
        
         checkboxChanged: function (evt) {
             
-            $cb      = $(evt.target),
+            var $cb      = $(evt.target),
             name     = $cb.attr('name');
+            //var checked = ($('#sites-table-div input').is(':checked'));
```
Comment (mbucknell) [inline nwc/src/main/webapp/js/view/AquaticBiologySelectFeaturesView.js]: Remove commented out code.

File: nwc/src/main/webapp/js/view/AquaticBiologySelectFeaturesView.js [hunk 2]
```diff
@@ -42,11 +43,15 @@ NWC.view.AquaticBiologySelectFeaturesView = NWC.view.BaseView.extend({
                             this.model.set({ 'selected' : selected});        
                         }
                     }
-            }
+            } //else {
+              //  $('.sites-table td input[type="checkbox"]').each(function() {
+              //          $(this).prop('checked', checked).change();
+              //      });
+            //}
```
Comment (mbucknell) [inline nwc/src/main/webapp/js/view/AquaticBiologySelectFeaturesView.js]: Remove commented out code.

**Comment by mbucknell** [comment]
:+1:

**Comment by kmschoep-usgs** [comment]
found the commented-out router code, too.

**Pull request merged**